wardrobe-auth/data/token.go

```diff
@@ -0,0 +1,6 @@
+package data
+
+type Tokens struct {
+	JWTToken     string `json:"jwt_token"`
```

Comment:
Странное название. JWT... не описывает назначение этого токена, а лишь указывает на его формат
```suggestion
	AccessToken     string `json:"access_token"`
```